roles/cifmw_cephadm/templates/cert_ceph_rgw.yml.j2

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ cifmw_cephadm_cert_name }}
+  namespace: {{ cifmw_cephadm_ns }}
+spec:
+  dnsNames: {{ cifmw_cephadm_cert_dns_names | list }}
```

Comment:
do we need something like:
```yaml
dnsNames:
{% for host in _hosts %}
  - {{ host }}
{% endfor %}
```
I assume the `cifmw_cephadm_cert_dns_names` can be automatically expanded here but I'm wondering if we want to compute the list of hosts (based on the inventory) or just provide this variable with a default || as an override.

Reply:
Yes, good idea. I've done that. thanks.